include/librealsense2/h/rs_option.h

```diff
@@ -171,6 +172,14 @@ extern "C" {
     } rs2_ambient_light;
     const char* rs2_ambient_light_to_string(rs2_ambient_light preset);
 
+    /** \brief digital gain for RS2_OPTION_DIGITAL_GAIN option. */
+    typedef enum rs2_digital_gain
+    {
+        RS2_DIGITAL_GAIN_HIGH_GAIN = 1,
```

Comment:
Can't we just use `DIGITAL_GAIN_HIGH` and `_LOW`?
We can change the text manually in the string function

Reply:
Although... just having `High` and `Low` appearing in the combo is fine with me, too...

Reply:
Enum renamed to low/high, The viewer will still display Low Gain / High Gain (unless you want it renamed too)